src/mem.rs

```diff
@@ -91,6 +91,10 @@ impl<T, const MUT: bool> Ptr<T, MUT> {
     pub fn is_null(self) -> bool {
         self.to_bits() == 0
     }
+
+    pub fn cast_void(self) -> Ptr<std::ffi::c_void, MUT> {
```

Comment:
This is a good idea and fits the convention of never importing `std::ffi::c_void`, thanks for adding it.

Reply:
No problem!